TracksDemo/TracksDemo/TracksDemo-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```

Comment:
Are we ever going to need a bridging header in this demo project? If not, should we remove this?

Reply:
I figured as long as the app is written in Obj-C we might as well keep it around. If we refactor it to Swift it can go then I was thinking 🤷

Reply:
👍